replay: alternative structure (#5913)
getsentry/sentry-docs: src/platforms/javascript/common/session-replay/privacy.mdx

---
title: Privacy
sidebar_order: 4200
description: "Configuring Session Replay to maintain user and data privacy."
---

> Sentry's Replay product is geared toward developers. By default, it'll mask all fiends, so no PII (personally identifiable information) is collected, eliminating any concerns with GDPR.

Author: The idea here was some general, more product-focus' spiel about how we see privacy. And that we default in a way that ppl can trust and use this thing. We could even talk about server side PII stripping here and link to docs

> Sentry's Replay product is geared toward developers. By default, it'll mask all fiends, so no PII (personally identifiable information) is collected, eliminating any concerns with GDPR.

Reviewer: ```suggestion
Sentry's Replay product is geared toward developers. By default, it'll mask all fiends, so no personally identifiable information (PII) is collected, eliminating any concerns with General Data Protection Regulation (GDPR).
```

> There are several ways to deal with PII. By default, the integration will mask all text content with `*` and block all media elements (`img, svg, video, object, picture, embed, map, audio`). This can be disabled by setting `maskAllText` to `false`. It is also possible to add the following CSS classes to specific DOM elements to prevent recording its contents: `sentry-block`, `sentry-ignore`, and `sentry-mask`. The following sections will show examples of how content is handled by the differing methods.

Reviewer: ```suggestion
There are several ways to deal with PII. By default, the integration will mask all text content with `*` and block all media elements (`img, svg, video, object, picture, embed, map, audio`). This can be disabled by setting `maskAllText` to `false`. It's also possible to add the following CSS classes to specific DOM elements to prevent recording their contents: `sentry-block`, `sentry-ignore`, and `sentry-mask`. The following sections will show examples of how content is handled by the differing methods.
```
